.changeset/lovely-bears-join.md

```diff
@@ -0,0 +1,6 @@
+---
+'@backstage/plugin-tech-insights-backend': minor
+'@backstage/plugin-tech-insights-node': minor
+---
+
+Propagate IdentityApi to the FactRetrieverContext so that FactRetrievers could invoke other plugins
```

Comment:
If we go ahead with this change I feel it's best to break this up into two changesets, and to mark the backend one with `**BREAKING:** `